cmd/eval-dev-quality/cmd/config.go

```diff
@@ -42,6 +59,16 @@ func (c *EvaluationConfiguration) Write(writer io.Writer) error {
 	return nil
 }
 
+// ReadEvaluationConfiguration reads an evaluation configuration file.
+func ReadEvaluationConfiguration(reader io.Reader) (configuration EvaluationConfiguration, err error) {
```

Comment:
The configuration holds a lot of data, plus it might be more in the future. Please return a pointer so we avoid copying all of it...